src/Files.App/Views/MainPage.xaml

```diff
@@ -132,24 +132,20 @@
 			Source="{x:Bind ViewModel.AppThemeBackgroundImageSource, Mode=OneWay}"
 			Stretch="{x:Bind ViewModel.AppThemeBackgroundImageFit, Mode=OneWay}" />
 
-		<Grid x:Name="RightMarginGrid" VerticalAlignment="Top">
+		<!--  Header Area  -->
+		<Grid>
 			<Grid.RowDefinitions>
 				<RowDefinition Height="Auto" />
 				<RowDefinition Height="Auto" />
 			</Grid.RowDefinitions>
 
-			<!--  Tab Control  -->
+			<!--  Tab Control (Lazy Load)  -->
```

Comment:
What do you mean by "Lazy Load"?

Reply:
To boost startup loading, d2dyno get almost of controls in MainPage to be loaded after a bit passed. You can see x:Load is False by default. The code behind shows them by finding these Xaml element.

Reply:
I see

Reply:
I'm not sure how helpful the comment is in the current form, do you think it needs to be mentioned at all?

Reply:
For me, felt in that way because it is unloaded at first and load afterwards and thought some people would wonder why.
But I also get to see the future of many people getting wondered what the Lazy Load means.

I think it's clearer to write all this down into the code behind instead. I'll revert.

Reply:
Done